wicketd/src/update_tracker.rs

```diff
@@ -919,9 +981,14 @@ impl UpdateDriver {
                 } => {
                     ensure!(id == update_id, "SP processing different update");
                     self.set_current_update_state(
-                        UpdateStateKind::ArtifactUpdateProgress {
-                            bytes_received: bytes_received.into(),
+                        UpdateStateKind::ArtifactWriteProgress {
+                            // We currently don't do any retries, so this is attempt 1.
+                            attempt: 1,
```

Comment:
Are we starting attempt counts at 1 or 0? (I would have assumed 0 but can understand 1)

Reply:
I was starting them at 1. I have a mild preference for 1 since it makes more sense intuitively and we aren't going to be doing modulo operations on them (best argument for starting an index at 0).

Reply:
Yeah, works for me. Starting at 1 means they can be displayed directly in the UI too.

Reply:
Yeah, that's what I was thinking.